spot-oa/oa/components/iana/dns-qclass.csv

```diff
@@ -6,5 +6,6 @@ Decimal,Hexadecimal,Name,Reference
 Technology Artificial Intelligence Laboratory, June 1981.]"
 4,0x00000004,Hesiod (HS),"[Dyer, S., and F. Hsu, ""Hesiod"", Project Athena Technical
 Plan - Name Service, April 1987.]"
+5,unknown,UNKNOWN,
```

Comment:
so why this particular value, instead of something that more explicitly says "this is a bad record that came from a null record"